assignment-5/server.js

```diff
@@ -0,0 +1,24 @@
+var express = require('express');
```

Comment:
@JainVayam Some points you need to remember when using the modules - 
- Require should be the first statement of the file. (ALL require statements should be written in first order)
- Please new lines.
- Again variable mistakes. What is the meaning of getApp, deleteApp, updateApp & insertApp? Instead the name should be like getUser, deleteUser, updateUser & insertUser. (Please follow the rule)